docs/source/usage/workflows/ionization.rst

```diff
@@ -0,0 +1,187 @@
+                ionizationEnergies<ionization::energies::AU::Nitrogen_t >               // <-- from step 2
+                massRatio<MassRatioNitrogen>,                                           // <-- from step 3
+                chargeRatio<ChargeRatioNitrogen>,                                       // <-- from step 3
+                ionizers<MakeSeq_t<               // <-- comma separated list of all ionizers of this species with
+                                                  // species of electrons macro particles to be created upon ionization
+                    particles::ionization::BSIEffectiveZ<Electrons, particles::ionization::current::PARAM_IONIZATIONCURRENT>,
+                    particles::ionization::ADKLinPol<Electrons, particles::ionization::current::PARAM_IONIZATIONCURRENT>,
+                    particles::ionization::ThomasFermi<Electrons>>>>;
+
+            using ParticleAttributesNitrogen = MakeSeq_t<position<position_pic>, momentum, weighting, boundElectrons>;
+            using Nitrogen = Particles<PMACC_CSTRING("N"), ParticleFlagsNitrogen, ParticleAttributesNitrogen>;
+        } // namespace picongpu
+
+    .. note::
+        Remember that you can define multiple electron species and associate them with your different ion species to track the ionisation processes separately!
+
+6. Create ion macro particles in the simulation and initialize them in :code:`speciesInitialization.param`.
+
+    .. code-block:: c++
+
+        namespace picongpu::particles
+        {
+            /* create nitrogen charge state 1 and electrons to neutralize the simulation */
+            using InitPipeline = pmacc::mp_list<
+                CreateDensity< ... some densityProfile... , startPosition::Random, Nitrogen>,
+                ManipulateDerive<manipulators::binary::DensityWeighting, Nitrogen, Electrons>,
+                Manipulate<manipulators::unary::ChargeState<1u>;, Nitrogen>>;
+        } // namespace picongpu::particles
+
+    .. attention::
+        Remember to initialize the simulation charge neutral to avoid creating ghost background charge densities.
+
+.. note::
+    For an example of a picongpu setup with ionization see the `FoilLTC example <https://github.com/ComputationalRadiationPhysics/picongpu/tree/dev/share/picongpu/examples/FoilLCT>`_.
+
+.. [1]
+    Clementi, E.; Raimondi, D. L. (1963)
+    "Atomic Screening Constants from SCF Functions"
+    J. Chem. Phys. 38 (11): 2686-2689.
+    https://dx.doi.org/10.1063/1.1733573
+.. [2]
+    Clementi, E.; Raimondi, D. L.; Reinhardt, W. P. (1967)
+    "Atomic Screening Constants from SCF Functions. II. Atoms with 37 to 86 Electrons"
+    Journal of Chemical Physics. 47: 1300-1307
+    https://dx.doi.org/10.1063/1.1712084
+.. [3]
+    Kramida, A., Ralchenko, Yu., Reader, J., and NIST ASD Team (2014)
+    NIST Atomic Spectra Database (ver. 5.2), [Online]
+    Available: http://physics.nist.gov/asd [2017, February 8]
```

Comment:
Optional, perhaps use the DOI instead of the direct URL:
```suggestion
    Available: https://dx.doi.org/10.18434/T4W30F [2017, February 8]
```

Reply:
done